abel/classes/interstage/plasma_lens/impactx.py

```diff
@@ -101,8 +135,10 @@ def get_impactx_lattice(self):
 
         # add lens offset for ISR mitigation
         if self.enable_isr and self.cancel_isr_kicks:
+            #self._isr_on_ref_part = True
             dx_isr = self.lens_offset_isr_kick_mitigation()
         else:
+            #self._isr_on_ref_part = False
```

Comment:
Per Kyrre's comment:
```suggestion
            dx_isr = self.lens_offset_isr_kick_mitigation()
        else:
```